### PR TITLE
qa: fixed tests 025, 151, 1041, 1215, 1478, 1483, 1484

### PR DESCRIPTION
**Pull request by kurik**
qa/025 - _wait_for_pmlogger() should append to $seq_full only if $seq_full is set _wait_for_pmlogger() in common.check appends success diagnostics to $seq_full. When that variable is empty, ">>$seq_full" becomes with no target, which produces shell errors i.e. in test 025 (cannot create : Directory nonexistent) polluting test output.

qa/1041 - propagate PCPQA_CHECK_DELAY to the sudo session PCPQA_CHECK_DELAY is not propagated to the "sudo ./Install" session, so it waits only for the default 1s instead of 10s. The fix implements propagation of the PCPQA_CHECK_DELAY. Reworked requirements when the test runs or is skipped.

qa/1215 - propagate PCPQA_KILL_SIGNAL to the sudo session The PCPQA_KILL_SIGNAL is not propagated to sudo session, so the test output expects HUP signal, but the default TERM is received instead. The fix propagates the PCPQA_KILL_SIGNAL to the sudo seesion, so HUP signal is used instead.

qa/1478 - propagate PCP_DIR and PCP_LOG_DIR to the sudo session The test builds an isolated config under $tmp. sudo does not preserve PCP_DIR or PCP_LOG_DIR, so those invocations used the system /etc/pcp.conf instead of $tmp/etc/pcp.conf. The fix propagates PCP_DIR and PCP_LOG_DIR to the sudo session

qa/1483 - fix filtering of syslog/journal noise

qa/1484 - propagate PCP_LOG_DIR to the sudo session pcp-reboot-init must run as root, so the test uses sudo. sudo does not preserve PCP_LOG_DIR, so the script used the system log directory (/var/log/pcp/...) instead of $tmp/log. The fix propagates the PCP_LOG_DIR to the sudo session

qa/151 - fix timing edge case when the test runs around midnight When the test runs soon after midnight (as in CI), “5 days ago at 00:10” can be only ~5.0 days old and survive the cull, while “6 days ago at 00:10” is ~6.0 days old and is removed. That matches the report: only TODAY-6 culled, TODAY-5 kept. The fix handles this specific edge case.

### CODE REVIEW

File: qa/1041 [hunk 1]
```diff
@@ -34,10 +34,28 @@ $python -c "import libvirt" >/dev/null 2>&1
...
+    || _notrun "no libvirt socket, libvirtd not running?"
+
+$python -c "
+import libvirt
+conn = libvirt.open('qemu:///system')
+if conn is None:
+    raise SystemExit(1)
+conn.close()
+" >/dev/null 2>&1 \
+    || _notrun "no libvirt qemu:///system connection driver available"
```
[the remark this answers is not in the document]
Comment (kurik) [reply qa/1041:58]: Fixed.

**Comment by kurik** [comment]
I am closing this PR. It is completely wrong. The problem is not in tests not properly propagating environment to `sudo` sessions. The problem is `ubuntu-26.04-container` using `sudo-rs`, not the classical `sudo`.
